recipes/vphaser2/meta.yaml

```diff
@@ -22,3 +23,8 @@ requirements:
 test:
     commands:
         - 'vphaser2 | grep "BAM file"'
+extra:
```

Comment:
I think it is better to use this URL: https://github.com/broadinstitute/v-phaser2/archive/6b5483a8e1a9bf24061a0aa211fee18e6b845668.tar.gz and keep the linter.

Reply:
well, I only updated license part, I did not modify build part, but indeed refering to an archive would be better for reproducibility. I gonna update recipe to do so

Reply:
added archive instead of git_url, removing linter excludes